release-notes/9.0/preview/preview2/api-diff/Microsoft.NETCore.App/9.0-preview2_System.IO.md

```diff
@@ -0,0 +1,10 @@
+# System.IO
+
+``` diff
+ namespace System.IO {
+     public class FileStream : Stream {
++        public override void CopyTo(Stream destination, int bufferSize);
```

Comment:
This is not new, the last time it was touched was 3 years ago:

![image](https://github.com/dotnet/core/assets/6011991/f5bdf9f6-704e-4ed2-a626-ebc0a5d4f3c9)

Reply:
Was it in the ref?

Reply:
@stephentoub Yes, at least since since 2020:

![image](https://github.com/dotnet/core/assets/6011991/9649b79f-898a-42aa-b33f-d07d70148b7b)

BTW GH does not like the size of this file:

![image](https://github.com/dotnet/core/assets/6011991/3f6f9f47-ef92-42d1-959d-c73e5a1e523b)

Reply:
https://github.com/dotnet/runtime/pull/97609/files#diff-cec8e6f471b4193246bdc0107b0dd7cbe131fb7fd189b288b37269c333d1171dR9923

Reply:
> ![image](https://private-user-images.githubusercontent.com/6011991/313115063-3f6f9f47-ef92-42d1-959d-c73e5a1e523b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA1MTg1MzUsIm5iZiI6MTcxMDUxODIzNSwicGF0aCI6Ii82MDExOTkxLzMxMzExNTA2My0zZjZmOWY0Ny1lZjkyLTQyZDEtOTU5ZC1jNzNlNWExZTUyM2IucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMxNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMTVUMTU1NzE1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YzgwMDRmOWY1YTMxOWZlMzI0NTkzYWUzZDNhM2NmNjk1ZWIzNGE0MjA1NmM4MWUxMDYyZGM4ODFhZTdmNDcxZCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.aI1l_S6_LqQJCfS2x9sPTm7YI-WRBNUiFsAStSolQK4)

[Insert moral philosophy joke here]